unitylibs/scripts/utils.js

```diff
@@ -68,7 +68,7 @@ export function getLocale() {
   const urlLocale = window.location.pathname.split('/')[1];
   const { locales } = getConfig();
   const locale = Object.entries(locales).find(([key]) => key === urlLocale);
-  return locale ? locale[1].ietf : 'en-US';
+  return locale ? locale[0] : 'us';
```

Comment:
Hi @Ruchika4 In place of splitting the url to find the locale and parsing through the object could we use the getConfig().locale? This property returned by config has the correct entry of the lcoale from the current url and can return the required property